lib/media/media_source_engine.js

```diff
@@ -211,6 +211,7 @@ shaka.media.MediaSourceEngine = class {
 
       return mediaSource;
     } else {
+      this.video_.disableRemotePlayback = false;
```

Comment:
My only concern is that we're changing the DOM now.  In general, if an app sets disableRemotePlayback to true in the DOM, we shouldn't change that here.

Is the default for disableRemotePlayback true or false?

Reply:
Default is `false`. We're explictly setting it to `true` when `ManagedMediaSource` is in use, I saw otherwise there are some problems even with rendering of demo page.

Reply:
We might stop setting it explicitly at all, but I think it may raise issues when `ManagedMediaSource` is used.

Reply:
OK, sounds good, then.